components/Common/Blockquote/index.tsx

```diff
@@ -0,0 +1,24 @@
+import type { ComponentProps, FC, PropsWithChildren } from 'react';
+
+import styles from './index.module.scss';
+
+type BlockquoteProps = {
+  attribution?: string;
+} & ComponentProps<'blockquote'>;
+
+const Blockquote: FC<PropsWithChildren<BlockquoteProps>> = ({
+  children,
+  cite,
+  attribution,
+}) => (
+  <blockquote className={styles.wrapper} cite={cite}>
+    {children}
+    {attribution && (
```

Comment:
I don't recall attribution being part of the blockquote component spec. I don't even think MDX can actually map that.